Tests/Core/EnvironmentTests.swift

```diff
@@ -0,0 +1,39 @@
+import XCTest
+
+@testable import Frames
+
+final class EnvironmentTests: XCTestCase {
+    
+    // MARK: - classicURL
+    
+    func test_classicURL_live_returnsCorrectValue() {
+        
+        let expectedURLString = "https://api2.checkout.com/v2/"
+        let actualURLString = Environment.live.classicURL.absoluteString
+        XCTAssertEqual(expectedURLString, actualURLString)
```

Comment:
is there a reason why we are comparing `String`s representing `URL`s instead of just comparing `URL`s

Reply:
I'll make this change 👍